mmcv/cnn/rfsearch/operator.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+import logging
+from typing import Dict
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from .utils import expands_rate, value_crop
+
+logger = logging.getLogger('Operators')
+logger.setLevel(logging.INFO)
+
+
+class ConvRFSearchOp(nn.Module):
+
+    def __init__(self, op_layer: nn.Module, global_config: Dict = {}):
+        """Based class of ConvRFSearchOp.
+
+        Args:
+        op_layer (nn.Module): pytorch module, e,g, Conv2d
+        global_config (Dict): config dict. Defaults to None.
+        """
```

Comment:
The docstring of __init__ need to be put between `class ConvRFSearchOp` and `def __init__`.